src/js/edu-benefits/5490/config/form.js

```diff
@@ -522,6 +522,33 @@ const formConfig = {
             educationProgram: {
               name: {
                 'ui:title': 'Name of school, university, or training facility you want to attend'
+              },
+              educationType: {
+                'ui:options': {
+                  updateSchema: (pageData, form, schema) => {
+                    // TODO: Reorder fields (maybe)...unless it's a universal change...
+                    // https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1794
+
+                    const newSchema = _.cloneDeep(schema);
+                    const benefitData = _.get('benefitSelection.data.benefit', form);
+                    const relationshipData = _.get('applicantInformation.data.relationship', form);
+
+                    // Remove tuition top-up
+                    const filterOut = ['tuitionTopUp'];
+                    // Correspondence not available to chapter35 children
+                    if (benefitData === 'chapter35' && relationshipData === 'child') {
+                      filterOut.push('correspondence');
+                    }
+                    // Flight training available to fry scholarships only
+                    if (benefitData !== 'chapter33') {
+                      filterOut.push('flightTraining');
```

Comment:
I think it was an either or, but @J-Quag, could you verify? If we only add the option in the correct scenario, do we need to edit the label?